### PR TITLE
Renaming 1a_gcloud_rolebinding.yaml and GCLOUD_SERVICEACCOUNT_CLIENT_EMAIL

### PR DESCRIPTION
**Pull request by imrajdas**
- Renaming 1a_gcloud_rolebinding.yaml to 1a_rolebinding.yaml 
- Renaming GCLOUD_SERVICEACCOUNT_CLIENT_EMAIL to SERVICEACCOUNT_CLIENT_EMAIL

### CODE REVIEW

**Comment by imrajdas** [comment]
cc @geekodour DCO passed

**Comment by geekodour** [comment]
@weastel could you see if this does not conflict with the unpushed work on the eks docs?

**Comment by weastel** [comment]
> @weastel could you see if this does not conflict with the unpushed work on the eks docs?

No, It will not create any conflict as eks does not need rolebinding to work unlike gke.

File: prombench/README.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ export GITHUB_REPO=prometheus
 ../infra/infra gke resource apply -a $AUTH_FILE -v PROJECT_ID:$PROJECT_ID -v ZONE:$ZONE \
     -v CLUSTER_NAME:$CLUSTER_NAME -v DOMAIN_NAME:$DOMAIN_NAME \
     -v GRAFANA_ADMIN_PASSWORD:$GRAFANA_ADMIN_PASSWORD \
-    -v GCLOUD_SERVICEACCOUNT_CLIENT_EMAIL:$GCLOUD_SERVICEACCOUNT_CLIENT_EMAIL \
+    -v SERVICEACCOUNT_CLIENT_EMAIL:SERVICEACCOUNT_CLIENT_EMAIL \
```
Comment (geekodour) [inline prombench/README.md]:
```suggestion
    -v SERVICEACCOUNT_CLIENT_EMAIL:$SERVICEACCOUNT_CLIENT_EMAIL \
```

Comment (imrajdas) [reply prombench/README.md]: Done

**Comment by imrajdas** [comment]
@geekodour Is this PR good?

**Pull request merged**